src/hostname/hostnamed.c

```diff
@@ -801,7 +801,9 @@ static int run(int argc, char *argv[]) {
                 return r;
 
         umask(0022);
-        mac_selinux_init();
+
+        if (mac_selinux_init() < 0)
+                log_warning("Failed to initialze SELinux support");
```

Comment:
typo

Reply:
hmm, so in some of our uses this function failing is fatal for the whole process, in others not. I figure we should clean this up, and have the same behaviour everywhere (probably have this fatal everywhere)

Reply:
hmm, actually mac_selinux_init() seems to log on its own about all errors. Hence I figure the caller should not log anymore. Otherwise we'd have duplicate logging.

Reply:
Reworked all `mac_selinux_init()` failures as fatal